### PR TITLE
Alembic migrations

### LINKED ISSUE #18: location Migration

### LINKED ISSUE #26: comment_like Migration

### LINKED ISSUE #27: badge Migration

### LINKED ISSUE #28: user_has_badge Migration

### PR DESCRIPTION
**Pull request by mattiasbarth**
Resolves #18
Resolves #26 
Resolves #27 
Resolves #28

### CODE REVIEW

File: app/Data/migrations/versions/126e2d6da815_location_table.py [hunk 1]
```diff
@@ -17,8 +18,20 @@
 
 
 def upgrade():
-    pass
+    op.create_table(
+        'badge',
```
Comment (AntonMaxen) [inline app/Data/migrations/versions/126e2d6da815_location_table.py:22]: Missing Foreignkeys, UserId and CategoryId

File: app/Data/migrations/versions/774ce7027729_comment_like_table.py [hunk 2]
```diff
@@ -17,8 +18,17 @@
 
 
 def upgrade():
-    pass
+    op.create_table(
```
Comment (AntonMaxen) [inline app/Data/migrations/versions/774ce7027729_comment_like_table.py:21]: UserId and CommentId needs to have sa.Foreignkey value to link tables,
and not sure if sa.BOOLEAN is a function
try sa.Boolean instead

File: app/Data/migrations/versions/9d9a42875d36_badge_table.py [hunk 3]
```diff
@@ -17,8 +18,18 @@
 
 
 def upgrade():
-    pass
+    op.create_table(
```
Comment (AntonMaxen) [inline app/Data/migrations/versions/9d9a42875d36_badge_table.py:21]: Looks fine

File: app/Data/migrations/versions/d46f8a17b349_user_has_badge_table.py [hunk 4]
```diff
@@ -17,8 +18,16 @@
 
 
 def upgrade():
```
Comment (AntonMaxen) [inline app/Data/migrations/versions/d46f8a17b349_user_has_badge_table.py:20]: Missing foreignkey linking

**Review by AntonMaxen — CHANGES_REQUESTED** [review]

**Review by AntonMaxen — APPROVED** [review]

**Review by HenrikLiman — APPROVED** [review]

**Pull request merged**